plugins/ccgx/fu-ccgx-plugin.c

```diff
@@ -36,6 +37,7 @@ fu_ccgx_plugin_constructed(GObject *obj)
 	fu_plugin_add_firmware_gtype(plugin, NULL, FU_TYPE_CCGX_FIRMWARE);
 	fu_plugin_add_firmware_gtype(plugin, NULL, FU_TYPE_CCGX_DMC_FIRMWARE);
 	fu_plugin_add_device_gtype(plugin, FU_TYPE_CCGX_HID_DEVICE);
+	fu_plugin_add_device_gtype(plugin, FU_TYPE_CCGX_PURE_HID_DEVICE);
```

Comment:
what's Pure in reference to? Is there a better name?

Reply:
As in it uses HID reports for everything. Unlike the FU_TYPE_CCGX_HID_DEVICE which uses HID only to switch to another mode.
Not sure of a better name.

Reply:
How about native hid?

Reply:
I guess we want to know if it's specific to anything. e.g. is this Bizlink specific? would FU_TYPE_CCGX_BIZLINK_DEVICE be more representative?

Reply:
Yup, I think we definitely need a better name now. This looks ODM specific -- so ideas very welcome.

Reply:
I'm actually wondering if this belongs in a new directory too. It doesn't seem to use much from the CCGX plugin at all!

Reply:
I'll be using the ccgx firmware parsing. And some of the constants.

> Yup, I think we definitely need a better name now. This looks ODM specific -- so ideas very welcome.

The HID protocol is actually included in the cypress/infineon CCG3 SDK, though slightly different.

Reply:
Maybe VendorHid instead then? Or NativeHid?

Reply:
I renamed it to NativeHid